examples/KDA/FLA_KDA/cumsum.py

```
@@ -0,0 +1,468 @@
+            scale=scale,
+            cu_seqlens=cu_seqlens,
+            head_first=head_first,
+            output_dtype=output_dtype,
+            chunk_indices=chunk_indices,
+        )
+    else:
+        raise ValueError(
+            f"Unsupported input shape {g.shape}, "
+            f"which should be (B, T, H, D) if `head_first=False` "
+            f"or (B, H, T, D) otherwise",
+        )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)